/usr/bin/ld: cannot find -lcupsimage

<!---
When creating a bug report please:
- Verify first that your issue is not already reported on GitHub
- Test if the latest release and master branch are affected too.
- Provide a clear and concise description of what the bug is in "Bug report 
  summary" section.
- Try to provide as much information about your environment (OS distribution,
  running in container, etc.) as possible to allow us reproduce this bug faster.
- Write which component is affected. We group our components the same way our
  code is structured so basically:
  component name = dir in top level directory of repository
- Describe how you found this bug and how we can reproduce it. Preferable with
  a minimal test-case scenario. You can paste gist.github.com links for larger
  files
- Provide a clear and concise description of what you expected to happen.
-->

##### Bug report summary

Hi,
When installing the current version, I have the message `/usr/bin/ld: cannot find -lcupsimage`.

It seems `libcupsimage2-dev` is a necessary dependency, but was not added in the `detect required packages` step.

##### OS / Environment

ubuntu 16.04

##### Netdata version (ouput of `netdata -V`)

`netdata v1.12.0-22-nightly`

Thanks !

--- a/configure.ac
+++ b/configure.ac
@@ -443,7 +443,7 @@ if test "${enable_plugin_cups}" != "no" -a "${have_cups}" = "yes"; then
     AC_DEFINE([HAVE_CUPS], [1], [cups usability])
 
     CUPS_CFLAGS="${CUPS_CFLAGS} `$CUPSCONFIG --cflags`"
-    CUPS_LIBS="${CUPS_LIBS} `$CUPSCONFIG --image --libs`"
+    CUPS_LIBS="${CUPS_LIBS} `$CUPSCONFIG --libs`"
 
     OPTIONAL_CUPS_CLFAGS="${CUPS_CFLAGS}"
     OPTIONAL_CUPS_LIBS="${CUPS_LIBS}"

